src/Cms/License.php

```diff
@@ -372,8 +404,19 @@ public function register(): static
 			);
 		}
 
+		if ($this->isFreeAndLocal() === true) {
+			$response = $this->selfsign([
+				'activation' => date('Y-m-d H:i:s'),
+				'code'       => $this->code,
+				'date'       => date('Y-m-d H:i:s'),
+				'domain'     => $this->domain,
+				'email'      => $this->email,
+				'order'      => '12345678',
```

Comment:
Nitpick: I would use something like 'free' or 'develop' or maybe a random hash. It's super theoretical, but I don't think we can guarantee that this would never collide with a paddle order id.